modules/light-clients/08-wasm/types/vm.go

```diff
@@ -112,7 +117,7 @@ func wasmInstantiate(ctx sdk.Context, clientStore storetypes.KVStore, cs *Client
 // - the response of the contract call contains non-empty events
 // - the response of the contract call contains non-empty attributes
 // - the data bytes of the response cannot be unmarshaled into the result type
-func wasmSudo[T ContractResult](ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState, payload SudoMsg) (T, error) {
+func wasmSudo[T ContractResult](ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState, payload SudoMsg, cdc codec.BinaryCodec) (T, error) {
```

Comment:
Just a consistency nit, but usually the codec is one of the first arguments, right?

```suggestion
func wasmSudo[T ContractResult](ctx sdk.Context, cdc codec.BinaryCodec, clientStore storetypes.KVStore, cs *ClientState, payload SudoMsg) (T, error) {
```

Reply:
nice suggestion!